artifacts/example.org/example.org.package.yml

```diff
@@ -0,0 +1,19 @@
+      mediatype: image/png
+spec:
+  maturity: alpha
+  description: ''
+  apis:
+    - name: oidc
+      currentVersion: '1.0'
```

Comment:
Should it be 1.0.0, to allow for future patches to be published. Or will future patch replace the existing 1.0 publication ?